site/components/Provider/Provider.tsx

```diff
@@ -34,6 +35,7 @@ const connectors = connectorsForWallets([
     groupName: 'More',
     wallets: [
       argentWallet({ chains }),
+      exodusWallet({ chains }),
```

Comment:
Please remove these changes to the docs site.

Reply:
@DanielSinclair Done in https://github.com/rainbow-me/rainbowkit/pull/567/commits/e8458bea1471b745d33f885de242b6a30cf86413